src/microphysics/opac_phoebus/opac_phoebus.hpp

```diff
@@ -19,8 +19,112 @@
 
 using namespace parthenon::package::prelude;
 
+// singularity includes
+#include <singularity-opac/base/radiation_types.hpp>
+#include <singularity-opac/neutrinos/mean_opacity_neutrinos.hpp>
+#include <singularity-opac/neutrinos/mean_s_opacity_neutrinos.hpp>
+#include <singularity-opac/neutrinos/opac_neutrinos.hpp>
+#include <singularity-opac/neutrinos/s_opac_neutrinos.hpp>
+
 namespace Microphysics {
 
+using RadiationType = singularity::RadiationType;
+
+/// One class to contain all opacity objects and wrap the subset of calls we use in
+/// phoebus for convenience.
+// TODO(BRR) template on opacity/scattering opacity being gray, and if they are constexpr
+// if-away the table lookups for mean opacities
```

Comment:
Should this be in singularity-opac? My feeling is no... and that it belongs in phoebus but I'm curious your thoughts

Reply:
I think I'm actually mistaken in this comment -- the mean opacity table lookups are over rho, T, Ye, *not* frequency, and an opacity that is not only gray but also doesn't depend on `rho`, `T`, `Ye` probably isn't very interesting i.e. we won't use that on large problems where performance is critical. So I'm inclined to just delete this comment and not worry about the case of mean opacity lookups when opacities dont depend on `rho`, `T`, `Ye`